build.sh

```diff
@@ -401,6 +410,15 @@ build_one() {
                 INCLUDE_PATH="${root}/include" \
                 LIBRARY_PATH="${root}/lib" install
         ;;
+    libzip)
+	do_cmake \
+                -DBUILD_PKGCONFIG_FILES=ON \
+                -DBUILD_DOC=ON \
+                -DZLIB_DLL=ON \
+                -D_WIN32=ON
```

Comment:
Please wait, I run into an error, apparently due to pkgconfig 
"checking for LIBZIP... no
configure: error: Package requirements (libzip >= 0.11) were not met:"
Need to investigate.